smart-app-banner.css

```diff
@@ -10,7 +10,7 @@
 .smartbanner {
 	position: absolute;
 	left: 0;
-	top: 0;
+	top: -80;
```

Comment:
Hi @dfcreative , I tried auto and it did not work. I'm going to create a separate pull request with 80px.